### PR TITLE
Add kernel-dragonball-experimental to kata-deploy, kata-deploy-test, and the release

### PR DESCRIPTION
**Pull request by fidencio**
kata-deploy: Add kernel-dragonball-experimental target

As Chao Wu added the support for building the dragonball kernel as a new
experimental kernel, let's make sure we reflect that as part of the
kata-deploy build scripts.

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>

---

release: Add the dragonball-experimental kernel

Let's add the dragonball specific kernel, which takes advantage of
upcall, as part of the release tarball, so it can be used from the
release tarball / kata-deploy.

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>

---

kata-deploy-test: Ensure we build dragonball specific kernel

As the dragonball specific kernel is now part of the release, let's make
sure we build it as part of the kata-deploy-test action.

Fixes: #5859

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>

---

### CODE REVIEW

File: tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh [hunk 1]
```diff
@@ -103,6 +104,13 @@ install_kernel() {
 	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -f -v "${kernel_version}"
 }
 
+#Install kernel asset
```
Comment (studychao) [inline tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh:107]: maybe the comment here could specify that this is dragonball experimental guest kernel to separate from the comment from `install_kernel()`

**Review by studychao — COMMENTED** [review]
Thanks @fidencio ,only a small comment:)

File: tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh (hunk 1 above)
Comment (fidencio) [reply tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh]: Thanks, updated!

File: tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh [hunk 2]
```diff
@@ -103,6 +104,13 @@ install_kernel() {
 	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -f -v "${kernel_version}"
 }
 
+#Install dragonball experimental kernel asseet
```
Comment (studychao) [inline tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh:107]: `asseet` -> `asset`

a simple typo here

**Review by studychao — APPROVED** [review]
Hi @fidencio , LGTM except a simple typo here.

**Comment by liubin** [comment]
@fidencio `kernel-dragonball-experimental` is added in https://github.com/kata-containers/kata-containers/pull/6141/commits/b7f4e96ff39941dd97130e1b6f0166b830e8f655, but seems not triggered in https://github.com/kata-containers/kata-containers/actions/runs/4031256028

**Comment by fidencio** [comment]
> @fidencio `kernel-dragonball-experimental` is added in [b7f4e96](https://github.com/kata-containers/kata-containers/commit/b7f4e96ff39941dd97130e1b6f0166b830e8f655), but seems not triggered in https://github.com/kata-containers/kata-containers/actions/runs/4031256028.
> 
> Maybe the workflow only run code in main branch, but not the PR.

I'm not exactly sure why GitHub does this every now and then. :-(

**Comment by fidencio** [comment]
@liubin, turns out I forgot to add it as part of the "kata-deploy-push" action.  I added it now and it seems to be finally triggered there: https://github.com/kata-containers/kata-containers/actions/runs/4042034528/jobs/6949258186

**Review by liubin — APPROVED** [review]

**Pull request merged**